modules/lazy-images/js/lazy-images.js

```diff
@@ -96,40 +100,50 @@ var jetpackLazyImagesModule = function( $ ) {
 	 * @param {object} image The image object.
 	 */
 	function applyImage( image ) {
-		var theImage = $( image ),
-			srcset,
+		var srcset,
 			sizes,
-			theClone;
+			imageClone,
+			lazyLoadedImageEvent;
 
-		if ( ! theImage.length ) {
+		if ( ! image instanceof HTMLImageElement ) {
 			return;
 		}
 
-		srcset = theImage.attr( 'data-lazy-srcset' );
-		sizes = theImage.attr( 'data-lazy-sizes' );
-		theClone = theImage.clone(true);
+		srcset = image.getAttribute( 'data-lazy-srcset' );
+		sizes = image.getAttribute( 'data-lazy-sizes' );
+		imageClone = image.cloneNode();
 
 		// Remove lazy attributes from the clone.
-		theClone.removeAttr( 'data-lazy-srcset' ),
-		theClone.removeAttr( 'data-lazy-sizes' );
-		theClone.removeAttr( 'data-lazy-src' );
+		imageClone.removeAttribute( 'data-lazy-srcset' ),
+		imageClone.removeAttribute( 'data-lazy-sizes' );
+		imageClone.removeAttribute( 'data-lazy-src' );
 
 		// Add the attributes we want on the finished image.
-		theClone.addClass( 'jetpack-lazy-image--handled' );
-		theClone.attr( 'data-lazy-loaded', 1 );
+		imageClone.classList.add( 'jetpack-lazy-image--handled' );
+		imageClone.setAttribute( 'data-lazy-loaded', 1 );
 		if ( ! srcset ) {
-			theClone.removeAttr( 'srcset' );
+			imageClone.removeAttribute( 'srcset' );
 		} else {
-			theClone.attr( 'srcset', srcset );
+			imageClone.setAttribute( 'srcset', srcset );
 		}
 		if ( sizes ) {
-			theClone.attr( 'sizes', sizes );
+			imageClone.setAttribute( 'sizes', sizes );
 		}
 
-		theImage.replaceWith( theClone );
+		image.parentNode.replaceChild( imageClone, image );
 
 		// Fire an event so that third-party code can perform actions after an image is loaded.
-		theClone.trigger( 'jetpack-lazy-loaded-image' );
+		try {
+			lazyLoadedImageEvent = new Event( 'jetpack-lazy-loaded-image', {
+				bubbles: true,
+				cancelable: true
+			} );
+		} catch ( e ) {
+			lazyLoadedImageEvent = document.createEvent( 'Event' )
+			lazyLoadedImageEvent.initEvent( 'jetpack-lazy-loaded-image', true, true );
+		}
+
+		imageClone.dispatchEvent( lazyLoadedImageEvent );
```

Comment:
**Note**: This is a potential regression.

The original implementation uses the `jQuery.trigger` method which fires custom jQuery events and unifies event data into [jQuery Event objects](https://api.jquery.com/category/events/event-object/). Such events are proprietary to jQuery and can't be observed using native APIs, e.g. `addEventListener`.

The issue here is that any 3rd party code relying on `jetpack-lazy-loaded-image` being a proprietary jQuery event _might_ not work after this change, because the native event object is different.

Reply:
Ah, interesting.

This in Jetpack could be edited to use the non-jQuery event:

https://github.com/Automattic/jetpack/blob/1115c95b8c41c72dac653d20b71a0e1ec9875a7f/modules/theme-tools/compat/twentysixteen.php#L63

...but of course the question of 3rd-party code remains.

Reply:
@kienstra Looked into differences between the native an jQuery event system. Shouldn't be an issue after all. Could you please review my reasoning in the comment above https://github.com/Automattic/jetpack/pull/14886/files#r390905356?